internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -23,13 +23,11 @@ import (
 
 	"github.com/SUSE/skuba/pkg/skuba"
 	"github.com/pkg/errors"
-	"k8s.io/klog"
```

Comment:
Is this a separate small cleanup bundled in the same PR, or was it removed for another reason I'm not seeing?

Reply:
It's the same change, after removing the `crio.reset` here: https://github.com/SUSE/skuba/pull/553/files#diff-eb6411fdf43c95124210c3fbb0eef978L63